parachain/pallets/omni-account/src/lib.rs

```diff
@@ -104,48 +104,55 @@ pub mod pallet {
 
 		/// The event type of this pallet
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+
 		/// The origin that represents the off-chain worker
 		type TEECallOrigin: EnsureOrigin<<Self as frame_system::Config>::RuntimeOrigin>;
-		/// The maximum number of identities an id graph can have.
+
+		/// The maximum number of accounts that an AccountGraph can have
 		#[pallet::constant]
-		type MaxIDGraphLength: Get<u32>;
+		type MaxAccountGraphLength: Get<MemberCount>;
```

Comment:
What about `MaxMemberAccountsLength` or `MaxMemberAccounts`?

Reply:
Yes I forgot to change that (which caused a compile error), sorry